src/components/card/QuizCard.jsx

```diff
@@ -1,8 +1,13 @@
-import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import { HiArrowNarrowRight } from "../../assets/icons.js";
 
-export const CategoryCard = ({quiz}) => {
-  const {_id, title, questions, } = quiz
+export const QuizCard = ({ quiz }) => {
+  const navigate = useNavigate();
+  const { _id, title, image_src,  questions } = quiz;
+  const clickTakeQuizHandler =()=>{
```

Comment:
Here the indentation could be better
It Could be like
clickTakeQuizHandler = () => {